features/attached_status.feature

```diff
@@ -58,7 +58,8 @@ Feature: Attached status
         Then stdout matches regexp:
         """
         One moment, checking your subscription first
-        Updating package lists
+        Updating CIS Audit package list
+        Updating main package list
```

Comment:
As I said in another comment, I would not use `main` here as it indicates we are only updating the `main` pocket - I'd say something different, like `standard Ubuntu package lists` or something like this. In this case it will be more than one package list btw

Reply:
Renamed.